Functions/fromCartesian().adoc

```diff
@@ -0,0 +1,56 @@
+= fromCartesian()
+
+=== Description
+
+Converts location of a point specified in Cartesian coordinates into Polar coordinates.
+
+=== Syntax
+
+`fromCartesian(x, y)`
+
+=== Parameters
+
+`x`: the _X_-coordinate, (`float`) 
+
+`y`: the _Y_-coordinate, (`float`)
+
+=== Returns
+
+Nothing
+
+=== Example
+
+[source, cpp]
+----
+
+#include <Coordinates.h>
+
+/* Code correctness not verified. To be completed soon*/
+
+void setup() {
+  // put your setup code here, to run once:
+  Serial.begin(9600);
+}
+
+Coordinates point = Coordinates();
+
+void loop() {
+  float x, y;
+  x = point.getX(); // Read X-coordinate of the point
+  y = point.getY(); // Read Y-coordinate of the point
+  point.fromCartesian(x, y);
```

Comment:
```suggestion
  Serial.print("Cartesian X coordinate: ");
  Serial.println(x);
  Serial.print("Cartesian Y coordinate: ");
  Serial.println(y);
  point.fromCartesian(x, y);
  Serial.print("Polar X coordinate: ");
  Serial.println(x);
  Serial.print("Polar Y coordinate: ");
  Serial.println(y);
```
Since you call `Serial.begin()`, might as well use it, right? I suggest similar things be done in the other examples.

Reply:
Thanks for your valuable suggestions @per1234 ! Will incorporate the changes you suggested!

Reply:
"/cc @per1234 for clarification on this ": 
 In line 51 shouldn't it be `Serial.println(r)` instead of `Serial.println(x)` as we are converting Cartesian coordinates to Polar? (In the same way `Serial.println(phi)` in line 53.)

Reply:
Good catch. I didn't take the time to look at it carefully. Here's my suggested example:
```c++
#include <Coordinates.h>
Coordinates point = Coordinates();
void setup() {
  Serial.begin(9600);
  float x = 1;
  float y = 1;
  Serial.print("x: ");
  Serial.println(x);
  Serial.print("y: ");
  Serial.println(y);
  point.fromCartesian(x, y);
  Serial.print("r: ");
  Serial.println(point.getR());
  Serial.print("φ: ");
  Serial.print(point.getAngle());
  Serial.println(" rad");
}

void loop() {}
```